dev-docs/show-outstream-video-ads.md

```diff
@@ -67,13 +65,16 @@ To display an outstream video, two things are needed:
 
 Prebid.js will select the `renderer` used to display the outstream video in the following way:
 
-1. If a `renderer` is associated with the Prebid adUnit, it will be used to display any outstream demand associated with that adUnit.  Below, we will provide an example of an adUnit with an associated `renderer`. If that renderer is specified as backup only, it will only be used when no other renderer is found. 
-2. If no `renderer` is specified on the Prebid adUnit, Prebid will invoke the renderer associated with the winning (or selected) demand partner video bid.  Choosing a backup only renderer allows publishers to access demand with or without an attached renderer.
+1. If a `renderer` is associated with the Prebid adUnit's video mediaType, it will be used to display any outstream demand associated with that adUnit with a mediaType of "video".
```

Comment:
I would remove the word outstream here

Reply:
@patmmccann - this document is called "Show Outstream Video Ads", so one argument is that this page should be focused on that use case.

Perhaps that implies we need a new page that focuses on "Multiformat Renderer" support?